django/forms/widgets.py

```diff
@@ -78,10 +79,24 @@ def render_css(self):
         ] for medium in media])
 
     def absolute_path(self, path, prefix=None):
+        """
+        Return absolute path to static asset.
+
+        Both absolute and relative paths are accepted.
+        An absolute path will be returned unchanged, where an relative
+        path will be transformed into the staticfile storage absolute path.
+
+        :param path: Relative or absolute path to static asset.
+        :type path: str
+        :param prefix: Added only if ``django.contrib.staticfiles`` is not enabled.
```

Comment:
I can't actually find a use for it in the source code, and it's not (AFAICT) a publicly documented API, so maybe we could deprecate it?